src/index.html

```diff
@@ -11,12 +12,113 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+  <!-- #end region -->
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo-link"
+      >
+        <img
+          src="images/moyo_1.png"
+          alt="Moyo Logo"
```

Comment:
The `alt` attribute should describe the image content. 'Moyo Logo' is a good description, but it could be more specific. For example, 'Moyo Store Logo' would be a more descriptive alt text.